api/ws/server.go

```diff
@@ -33,16 +34,25 @@ var (
 	ErrExpired = errors.New("expired")
 )
 
-func WithWebsocketAPI(maxPendingMessages int) vm.Option {
-	return func(v *vm.VM) error {
+type WSConfig struct {
```

Comment:
nit: name this `Config`. In Go the package is the level of abstraction so it seems weird to have a `ws.WsConfig`.

Reply:
done